src/index.ts

```diff
@@ -1 +1,2 @@
 export { GoogleSheetsConnector } from './GoogleSheetsConnector'
+export { GoogleLanguageConnector } from './GoogleLanguageConnector'
```

Comment:
hmm I think I made a mistake here, we also need to export interfaces for config and event options.

can you convert both to:
export * from './file'

that should do the trick